tests/framework/utils/snapshot.go

```diff
@@ -40,6 +40,8 @@ const (
 // CheckSnapshotISReadyToUse checks snapshot is ready to use
 func (k8sh *K8sHelper) CheckSnapshotISReadyToUse(name, namespace string, retries int) (bool, error) {
 	for i := 0; i < retries; i++ {
+		// sleep first and try to check snapshot is ready to cover the error cases.
+		time.Sleep(time.Duration(i) * time.Second)
```

Comment:
@Madhu-1 in the first iteration i=0 so it will not sleep.

Reply:
Yes, that is the reason for adding this one. we don't need sleep for the first iteration if there are any errors in the first iteration then sleep will be added.

Reply:
okay :+1: